### PR TITLE
Move recipes from meta-intel-aero-connectivity

### PR DESCRIPTION
**Pull request by avinash-palleti**
Migrated recipes from intel-aero-connectivity layer to intel-aero. This
will make easy to find delta between yocto baseline and intel-aero.

Issue: https://github.com/intel-aero/meta-intel-aero/issues/43

Signed-off-by: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>

### CODE REVIEW

File: recipes-core/images/intel-aero-image.bb [hunk 1]
```diff
@@ -61,6 +61,18 @@ GRUB_TIMEOUT = "3"
 IMAGE_INSTALL += "librealsense"
 IMAGE_INSTALL += "librealsense-graphical-examples"
 
+#connectivity
+IMAGE_INSTALL += "backport-iwlwifi"
+IMAGE_INSTALL += "linux-firmware-iwlwifi-8000c"
```
Comment (lucasdemarchi) [inline recipes-core/images/intel-aero-image.bb]: great... now it's even clear the linux-firmware thing.  On the other PR, check the .dot file that is created... it should tell you who is bringing the entire linux-firmware package

File: recipes-core/images/intel-aero-image.bb [hunk 2]
```diff
@@ -61,6 +61,18 @@ GRUB_TIMEOUT = "3"
...
+IMAGE_INSTALL += "hostapd"
+IMAGE_INSTALL += "autostart-hostapd"
+IMAGE_INSTALL += "autostart-supplicant"
+
+# LTE MODEM
+IMAGE_INSTALL += "glibc-gconvs glibc-utils glibc-gconv-iso8859-1 modemmanager \
+	opkg opkg-utils rpm icon-naming-utils libtool libndp libnl libinput \
```
Comment (lucasdemarchi) [inline recipes-core/images/intel-aero-image.bb]: opkg opkg-utils rpm? the opkg packages must be wrong.  And these libraries should be dependency from the tools, no need to add them here

File: recipes-support/modem-enable/modem-enable.bb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    install -d ${D}${sysconfdir}/rc5.d/
+    install -d ${D}${sysconfdir}/rc6.d/
+    install -m 0755 ${WORKDIR}/modem-enable ${D}${sysconfdir}/init.d/
+    ln -sf ../init.d/modem-enable      ${D}${sysconfdir}/rc0.d/K20modem-enable
+    ln -sf ../init.d/modem-enable      ${D}${sysconfdir}/rc1.d/K20modem-enable
+    ln -sf ../init.d/modem-enable      ${D}${sysconfdir}/rc2.d/S20modem-enable
+    ln -sf ../init.d/modem-enable      ${D}${sysconfdir}/rc3.d/S20modem-enable
+    ln -sf ../init.d/modem-enable      ${D}${sysconfdir}/rc4.d/S20modem-enable
+    ln -sf ../init.d/modem-enable      ${D}${sysconfdir}/rc5.d/S20modem-enable
+    ln -sf ../init.d/modem-enable      ${D}${sysconfdir}/rc6.d/k20modem-enable
```
Comment (lucasdemarchi) [inline recipes-support/modem-enable/modem-enable.bb]: Add a commit on top moving this to use rc-update. So it's easier to grasp and will allow us to test systemd without these files installed manually.

Comment (lucasdemarchi) [inline recipes-support/modem-enable/modem-enable.bb]: See: http://www.yoctoproject.org/docs/1.8/ref-manual/ref-manual.html#var-INITSCRIPT_PARAMS and http://www.yoctoproject.org/docs/1.8/ref-manual/ref-manual.html#var-INITSCRIPT_NAME and http://www.yoctoproject.org/docs/1.8/ref-manual/ref-manual.html#ref-classes-update-rc.d

File: recipes-core/images/intel-aero-image.bb (hunk 1 above)
Comment (avinash-palleti) [inline recipes-core/images/intel-aero-image.bb]: it is coming for packagegroup-machine-base

File: recipes-core/images/intel-aero-image.bb (hunk 2 above)
Comment (avinash-palleti) [inline recipes-core/images/intel-aero-image.bb]: Only opkg and opkg-utils ?
Do i need to run any test to make sure it is not breaking something else ?

File: recipes-support/modem-enable/modem-enable.bb (hunk 3 above)
Comment (avinash-palleti) [inline recipes-support/modem-enable/modem-enable.bb]: Can we do these changes as part of another PR as this PR having so many changes and hard to maintain for longer time :)

File: recipes-core/images/intel-aero-image.bb (hunk 2 above)
Comment (lucasdemarchi) [inline recipes-core/images/intel-aero-image.bb]: Let's keep those in our list of "possible things to remove" and not change it in this PR.

File: recipes-support/modem-enable/modem-enable.bb (hunk 3 above)
Comment (lucasdemarchi) [inline recipes-support/modem-enable/modem-enable.bb]: sure thing... feel free to create an issue on github, add these things and assign to you :)

Comment (avinash-palleti) [inline recipes-support/modem-enable/modem-enable.bb]: Done. 👍

Comment (avinash-palleti) [inline recipes-support/modem-enable/modem-enable.bb]: Done 👍

Comment (avinash-palleti) [inline recipes-support/modem-enable/modem-enable.bb]: Done :)

File: conf/bblayers.conf.sample [hunk 4]
```diff
@@ -14,7 +14,6 @@ BBLAYERS ?= " \
   ##OEROOT##/meta-uav \
   ##OEROOT##/meta-ros \
   ##OEROOT##/meta-intel-aero \
-  ##OEROOT##/meta-intel-aero-connectivity \
```
Comment (lucasdemarchi) [inline conf/bblayers.conf.sample]: @whbruce do we have to change POKY_BBLAYERS_CONF_VERSION above for this change?

Comment (whbruce) [inline conf/bblayers.conf.sample]: No need to bump this variable. POKY_BBLAYERS_CONF_VERSION refers to the bblayers schema, not the content so could only change between Yocto releases.
